.circleci/config.yml

```diff
@@ -218,9 +218,14 @@ jobs:
       - run:
           name: "Test: Build and Spin-up Django API Service"
           command: "cd tdrs-backend; docker-compose up -d --build"
+      - run:
+          name:  "Test: Execute Python Linting Test"
+          command: "cd tdrs-backend; docker-compose run --rm web bash -c \"flake8 .\""
       - run:
           name: "Test: Run Unit Tests And Create Code Coverage Report"
-          command: "cd tdrs-backend; docker-compose run --rm web bash -c \"pytest --cov-report=xml\""
+          command: |
+            cd tdrs-backend;
+            docker-compose run --rm web bash -c "./wait_for_services.sh && pytest --cov-report=xml"
```

Comment:
When launching the tests with `docker-compose run` the tests won't wait until all services have been launched since the command passed here overrides the docker entrypoint that usually handles this. To correct this we need to explicitly utilize the `wait_for_services.sh` script here.